workorders.lic

```diff
@@ -221,7 +221,8 @@ class WorkOrders
   def carve_items(info, item, quantity)
     ensure_copper_on_hand(@cash_on_hand || 5000, @settings)
     recipe, items_per_stock, spare_stock, scrap = find_recipe(info, item, quantity)
-
+    rock_noun = %w[deed pebble stone rock rock boulder]
```

Comment:
an extra rock here

Reply:
Yes intentional
Deed, pebble, stone, small rock, large rock, boulder

Reply:
❓ I'm not clear on how `rock_noun` and `rock_count` work. Isn't `stock-volume` assuming you get a `stone`?

If I'm reading the loop on line 249 correctly, the first time through the loop `count` is zero, so `rock_count = info['stock-volume']` (i.e. buy a `rock`). How does that account for when `recipe['volume']` requires something bigger than a rock (or allows for something smaller like a pebble`?

Reply:
No currently in crafting yaml its a small rock,
Volume 3. Currently no workorders in recipes are larger than 3 volume.